modules/nvidia-container-toolkit.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, ... }:
+with builtins; with lib; {
+
+  config =
+    mkIf (config.wsl.enable &&
+          (config.wsl.docker-desktop.enable ||
+           config.virtualisation.podman.enable ||
+           (config.virtualisation.docker.enable &&
+            (versionAtLeast config.virtualisation.docker.package.version "25")))) {
```

```suggestion
  config = mkIf (
    config.wsl.enable && (
      config.wsl.docker-desktop.enable ||
      config.virtualisation.podman.enable ||
      (config.virtualisation.docker.enable && (versionAtLeast config.virtualisation.docker.package.version "25"))
    )
  ) {
```
or something like that

---

Updated with `nixpkgs-fmt`.